### ISSUE TITLE
Multiple PayPal checkout buttons not appearing correctly on page

### ISSUE THREAD

**Issue by flyingL123** [paypal/paypal-checkout#890]
### Description

Our website is hosted with BigCommerce, and I have noticed recently that our PayPal buttons do not all appear as expected on the cart page. The cart page has 3 locations where the button is supposed to appear.

Location 1: Mobile-only top location
Location 2: Desktop-only top location
Location 3: Mobile and Desktop bottom location

The buttons used to appear in all three locations as expected, but recently I've noticed that only Location 1 is appearing, and the other two locations do not show a button.

This is troublesome for us since we need the button to appear in multiple places. For example, here is the mobile layout, which works correctly since the mobile-only top location is the first one in the markup. The PayPal button appears correctly underneath the AmazonPay button:

<img width="378" alt="paypal_location1" src="https://user-images.githubusercontent.com/2486712/46834010-2d06f380-cd78-11e8-8f34-9ebb4aff8fcd.png">

However, there should also be a PayPal button in Location 3 underneath the Amazon button, but as you can see, it does not appear:

<img width="378" alt="paypal_location3" src="https://user-images.githubusercontent.com/2486712/46834796-64769f80-cd7a-11e8-8ff7-47ce5483ed7d.png">

Additionally, on desktop, no PayPal buttons appear since desktop uses Location 2 and Location 3:

<img width="1190" alt="paypal_location2and3" src="https://user-images.githubusercontent.com/2486712/46834043-44de7780-cd78-11e8-9d17-97546f6c1e14.png">

Location 2 and 3 both contain the code to show the button, but they never appear:

<img width="1341" alt="paypal_code" src="https://user-images.githubusercontent.com/2486712/46834096-79523380-cd78-11e8-861f-cf9dbc581814.png">

**Screenshots or videos that show the issue**
Shown above in description
**Your code**
The code is injected onto the page via BigCommerce.
**The `paypal.version` from your browser console**
4.0.224
**The exact browser version**
Version 69.0.3497.100 (Official Build) (64-bit)
**A link to your page with the issue, if possible**
Go to https://www.storeyourboard.com/skateboard-deck-display-floating-mount/ and click "Add to Cart". The issue appears on the cart page as long as an item is in the cart.
**All console logs during the time of the issue, especially error messages**
None

**Does the issue also occur at [developer.paypal.com/demo/checkout](https://developer.paypal.com/demo/checkout)?**
Yes. If you replace line 11 with these two lines:
```
<div data-paypal-button-container></div>
<div data-paypal-button-container></div>
```
Then change line 52 to:
```
 }, '[data-paypal-button-container]');
```
You will see that only one button appears.

**Does the issue occur consistently, or intermittently?**
Consistently

### Steps to reproduce

Go to https://www.storeyourboard.com/skateboard-deck-display-floating-mount/ and click "Add to Cart". The issue appears on the cart page as long as an item is in the cart.

In other words, the checkout buttons only appear when an item is in your cart. So make sure there is an item in your cart. Then anytime you visit the cart page, you will see the issue.

### Affected browsers

- [x] Chrome
- [x] Safari
- [x] Firefox
- [ ] Edge
- [ ] IE
- [x] Chrome Mobile/Tablet
- [x] Safari Mobile/Tablet
- [ ] Web View / Safari ViewController
- [ ] Other

**Comment by flyingL123** [comment]
Thanks. I think you are correct and this may be an issue with BigCommerce's integration. Seems like they recently released a checkout SDK and I bet that's causing the problem. This is likely going to be a nightmare trying to get in touch with them to explain the issue to the right person. We'll see!
